resolwe_bio/processes/import_data/pathway_map.yml

```diff
@@ -0,0 +1,34 @@
+- slug: upload-metabolic-pathway
+  name: Metabolic Pathway File
+  requirements:
+    expression-engine: jinja
+    executor:
+      docker:
+        image: resolwebio/legacy:latest
+    resources:
+      network: true
+  data_name: Metabolic Pathway
+  version: 0.1.0
+  type: data:pathway
+  category: upload
+  persistence: RAW
+  scheduling_class: interactive
```

Comment:
I am not sure about this, but import processes should not have interactive `scheduling_class`, right, @mstajdohar ?

Reply:
Maybe they can if they are really, really fast. @dblenkus ?

Reply:
Interactive scheduling class means that UI depends on the results of the process. To make it efficient, such processes are limited to 30 seconds (excluding the upload). If this process meets both conditions then it can (and should) be interactive.

Reply:
Apart from upload this process is just saving a JSON to the DB, which is around 0.5 Mb. I think it can be done under 30s.

Reply:
Will this maps be continuously saved to DB (as per user action), or are they saved once and reused?

Reply:
Saved once and reused, so I guess making it non-interactive is also fine.

Reply:
In that case, it is better to make it non-interactive, so that it does not compete with all the interactive processes for resources.